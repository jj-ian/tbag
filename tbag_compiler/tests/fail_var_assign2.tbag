true {
        testvar();
        endgame;
}

func int testvar() {
        int i;
        string j = "hello";
        i = j; 
        return 0;
}
