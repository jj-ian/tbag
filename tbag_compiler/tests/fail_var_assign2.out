Fatal error: exception Failure("Type mismatch in assignment statement")
